### PR TITLE
Add note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3808

### CODE REVIEW

File: SharePoint/SharePointOnline/sync-health.md [hunk 1]
```diff
@@ -116,6 +116,9 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
 > [!IMPORTANT]
 > When you enable the EnableSyncAdminReports setting on devices, it can take up to three days for reports to be available. Devices will appear in the report after this time. Forcing a specific device to report data is unsupported.
 
+> [!NOTE]
+> OneDrive sync health reports can also be enabled using administrative templates in Intune, for more information, see [Use administrative templates in Intune](configure-sync-intune.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:120]:
```suggestion
> OneDrive sync health reports can also be enabled using administrative templates in Intune. For more information, see [Use administrative templates in Intune](configure-sync-intune.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**